kairon/chat/actions.py

```
@@ -239,5 +247,81 @@ async def run(
+        )
+
+        async with session:
+            async with session.request(
+                method,
+                url,
+                headers=headers,
+                params=endpoint_config.combine_parameters(kwargs),
+                compress=compress,
+                ssl=sslcontext,
+                **kwargs,
+            ) as response:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)